### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by aitorarjona**
- Update stats docs after #901
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: docs/source/api_stats.rst [hunk 1]
```diff
@@ -71,26 +71,26 @@ The user can obtain these statistics through the future object:
...
-     'worker_exec_time': 3.26375866,
-     'worker_func_end_tstamp': 1647258754.9280388,
-     'worker_func_exec_time': 3.00374055,
-     'worker_func_start_tstamp': 1647258751.9242983,
-     'worker_result_upload_time': 0.05767894,
-     'worker_start_tstamp': 1647258751.739544}
+     'host_status_done_tstamp': 1647526903.6858306,
+     'host_status_query_count': 2,
+     'host_submit_tstamp': 1647526901.3674674,
+     'warm_container': True,
```
Comment (JosepSampe) [inline docs/source/api_stats.rst:86]: @aitorarjona to align this metric to the others, could we change `warm_container` to `worker_cold_start` or so (as sometimes in standalone there are no containers)?

Comment (aitorarjona) [reply docs/source/api_stats.rst:86]: sure

File: docs/source/api_stats.rst [hunk 2]
```diff
@@ -120,11 +120,11 @@ The user can obtain these statistics through the future object:
    * - :code:`host_result_done_tstamp`
      - Timestamp of when host received the function result from cloud object storage.
    * - :code:`host_result_query_count`
-     -
+     - Number of list query to the object storage to get the status object (synchronize the completion of the function)
```
Comment (JosepSampe) [inline docs/source/api_stats.rst:123]: For now only "get queries" are count.  "list queries" are not count

File: docs/source/api_stats.rst [hunk 3]
```diff
@@ -120,11 +120,11 @@ The user can obtain these statistics through the future object:
...
    * - :code:`host_status_done_tstamp`
      - Timestamp of when the host received the signal that the function has finished its execution.
    * - :code:`host_status_query_count`
-     -
+     - Number of list query to the object storage to get the result object
```
Comment (JosepSampe) [inline docs/source/api_stats.rst:127]: The same as above. Only "get queries" are count

**Pull request merged**